rawler/data/lenses.toml

```diff
@@ -204,6 +211,13 @@ model = "150-600mm f/5-6.3 DG OS HSM | C"
 focal_range = [[150, 1], [600, 1]]
 aperture_range = [[50, 10], [63, 10]]
 
+[[lenses]]
+key = "18-300mm F3.5-6.3 DC MACRO OS HSM | Contemporary 014"
+make = "Sigma"
+model = "18-300mm F3.5-6.3 DC MACRO OS HSM | Contemporary 014"
```

Comment:
Correct model is "18-300mm f/3.5-6.3 DC Macro OS HSM | C"

Reply:
fixed

Reply:
Still wrong, it's f/3.5 instead of F3.5 :wink:

Reply:
woops... fixed